tests/panels/test_request.py

```diff
@@ -104,3 +104,22 @@ def test_namespaced_url(self):
         self.panel.generate_stats(self.request, response)
         panel_stats = self.panel.get_stats()
         self.assertEqual(panel_stats["view_urlname"], "admin:login")
+
+    def test_session_list_sorted_or_not(self):
+        """
+        Verify the session is sorted when all keys are strings.
+
+        See  https://github.com/jazzband/django-debug-toolbar/issues/1668
+        """
+        self.request.session = {
+            1: "value",
+            "data": ["foo", "bar", 1],
+            (2, 3): "tuple_key",
+        }
+        data = {
+            "list": [(1, "value"), ("data", ["foo", "bar", 1]), ((2, 3), "tuple_key")]
+        }
+        response = self.panel.process_request(self.request)
+        self.panel.generate_stats(self.request, response)
+        panel_stats = self.panel.get_stats()
+        self.assertEqual(panel_stats["session"], data)
```

Comment:
```suggestion
        self.assertEqual(panel_stats["session"], data)
        
        self.request.session = {
            "b": "b-value",
            "a": "a-value",
        }
        data = {
            "list": [("a", "a-value"), ("b", "b-value")]
        }
        response = self.panel.process_request(self.request)
        self.panel.generate_stats(self.request, response)
        panel_stats = self.panel.get_stats()
        self.assertEqual(panel_stats["session"], data)        
```

Reply:
i guess I should also test that in a new test

Reply:
I think it's fine to keep both in a single test (as suggested)

Reply:
@ritiksoni00 you can either copy this code into your local environment and commit or use github's "Commit suggestion" button to create a commit on the PR. When we merge it, we'll squash the commits into one.